PR: https://github.com/odoo/odoo/pull/

From: 8084f3658c663ce09456638e02c951b55785dba2
From: Simon Lejeune
Date: 2020-01-31 10:22:14

Structural Changes: 4
Total Changes: 6

[REF] stock: wording

This is not related to the batch picking task but was found while
testing it.

task-2069646

================================= pseudo patch: =================================

--- a/addons/stock/models/stock_move.py
+++ b/addons/stock/models/stock_move.py
@@ -54,7 +54,7 @@ class StockMove(models.Model):
         digits=0, store=True, compute_sudo=True,
         help='Quantity in the default UoM of the product')
     product_uom_qty = fields.Float(
-        'Initial Demand',
+        'Demand',
         digits='Product Unit of Measure',
         default=0.0, required=True, states={'done': [('readonly', True)]},
         help="This is the quantity of products from an inventory "
@@ -93,7 +93,7 @@ class StockMove(models.Model):
         'stock.move', 'stock_move_move_rel', 'move_dest_id', 'move_orig_id', 'Original Move',
         copy=False,
         help="Optional: previous stock move when chaining them")
-    picking_id = fields.Many2one('stock.picking', 'Transfer Reference', index=True, states={'done': [('readonly', True)]}, check_company=True)
+    picking_id = fields.Many2one('stock.picking', 'Transfer', index=True, states={'done': [('readonly', True)]}, check_company=True)
     picking_partner_id = fields.Many2one('res.partner', 'Transfer Destination Address', related='picking_id.partner_id', readonly=False)
     note = fields.Text('Notes')
     state = fields.Selection([

--- a/addons/stock/models/stock_move_line.py
+++ b/addons/stock/models/stock_move_line.py
@@ -15,7 +15,7 @@ class StockMoveLine(models.Model):
     _order = "result_package_id desc, id"
 
     picking_id = fields.Many2one(
-        'stock.picking', 'Stock Picking', auto_join=True,
+        'stock.picking', 'Transfer', auto_join=True,
         check_company=True,
         help='The stock operation where the packing has been made')
     move_id = fields.Many2one(
